### ISSUE TITLE
Firefox does not show first subtitle with splash setups

### ISSUE THREAD

**Issue by seizv** [flowplayer/flowplayer#1180]
If subtitles start in 0 sec, flowplayer does not show the first subtitles, but if seeking back manually to 0, flowplayer showed the first subtitle. This bug has been detected at firefox versions 52 and 51 in win7
It probably could be found in other versions and browsers too.

https://codepen.io/SeizV/pen/PmNxWx flowplayer 6.0.3
or
https://codepen.io/SeizV/pen/YVqMoX flowplayer 7.0.2

And subtitles does not show in safari 10.

https://youtu.be/ZdDjKwxNg68 'testing show first subtitle in browserstack'

**Comment by nnarhinen** [comment]
Please test with 7.0.3. Most likely a duplicate of #1164

**Comment by nnarhinen** [comment]
This is by definition - splash setup does not show any controls.

https://codepen.io/anon/pen/YVpzLx -- first frame setup

**Comment by seizv** [comment]
How can I use "splash" if I want the subtitles looks correct in FF and safari? The chrome displays subtitles correctly.

**Comment by blacktrash** [comment]
@seizv - at http://demos.flowplayer.org/basics/subtitle.html#vtt (splash) the subtitles are not shown in Chrome (are they for you?).
Subtitles are not part of a splash or poster image, they overlay a _video_ frame. Therefore, if the first frame is not shown (splash or poster state) they will not be shown either. Only once you start playback.

**Comment by seizv** [comment]
@blacktrash What about this https://codepen.io/SeizV/pen/YVqMoX. Chrome show first subtitle. Firefox does not show first subtitle, but show second subtitle, if seeking back manually to 0, firefox showed the first subtitle too.

**Comment by blacktrash** [comment]
`splash: true` means that the player is not loaded, it's independent of whether there's an image or background color, therefore no video frame is visible and also no subtitles, just a white area in your example.
When you seek back to 0, the first frame is visible, and therefore also the subtitles.

See https://flowplayer.org/docs/setup.html#splash and https://flowplayer.org/docs/setup.html#poster
If you want to *show* the first frame, splash must be false and no poster must be set, neither an image or background-color, as at https://codepen.io/anon/pen/YVpzLx

**Comment by nnarhinen** [comment]
Ok now I get it. Firefox (in splash setup) does not show the first subtitle at all.

**Comment by blacktrash** [comment]
Strange, could not repro with FF 53.0 and previous on Mac.

**Comment by nnarhinen** [comment]
I could with some setups. Tried to reproduce on codepen and couldn't anymore. You have to trust me with this one :D (and maybe try at demos.dev94.flowplayer.us that I did not break anything that used to work..)

**Comment by blacktrash** [comment]
Trust me to trust you :D
And at least the subtitle demos at dev94 are not broken.

**Comment by seizv** [comment]
@nnarhinen  I reproduce this bug in browserstack.com, select win7 and FF 53. 
https://codepen.io/SeizV/pen/gWryRb with flowplayer 7.0.4.
